docs/09-Advanced.md

```diff
@@ -34,6 +34,25 @@ myLineChart.data.datasets[0].data[2] = 50; // Would update the first dataset's v
 myLineChart.update(); // Calling update now animates the position of March from 90 to 50.
 ```
 
+#### .updateConfig(partialConfig) 
+
+Triggers an update of the chart config. This allows for dynamic updating of the config including the axes. It does NOT support changing the type of the chart. To do that, the chart must be recreated. This method does not update the chart. `.update()` should be used after this function is called.
+
+```javascript
+// Hide the title and legend from the chart
+myLineChart.updateConfig({
```

Comment:
Since you still have to call `update()` afterwards, could we call this `config()` or `setConfig()` instead?